packages/teams-js/test/internal/communication.spec.ts

```diff
@@ -6,7 +6,7 @@ import { NestedAppAuthMessageEventNames, NestedAppAuthRequest } from '../../src/
 import { ApiName, ApiVersionNumber, getApiVersionTag } from '../../src/internal/telemetry';
 import { UUID } from '../../src/internal/uuidObject';
 import { FrameContexts } from '../../src/public';
-import { app } from '../../src/public/app';
+import * as app from '../../src/public/app';
```

Comment:
Just testing my knowledge: there's no reason we *shouldn't* `import * from bla` instead of importing specific types, correct? I think tree shaking will be "smart" enough to still remove anything you aren't literally referencing in the file.

Reply:
Typically, it's best practice to name all of the imports you would want to use (provided they are named exports). A bundler with well implemented treeshaking _**should**_ be smart enough to shake out all unused functions making this syntax fine, but in theory there is a small possibility that they don't get treeshaken out since it is not something we can directly control. In this scenario it is perfectly fine since it's just a test file, and we're not running it through a bundler anyways nor are we particularly concerned with storage complexity. Also, by using this import syntax we can still treat all of the imports as `app.whatever`, so I don't have to update all of the function call references.

Reply:
Okay, makes sense! Just wanted to verify this was a pattern we were okay with people copying (and I think we are)